src/v/datalake/translation/partition_translator.cc

```diff
@@ -275,6 +275,7 @@ partition_translator::do_translate_once(retry_chain_node& parent_rcn) {
           read_begin_offset,
           read_end_offset,
           _partition->last_stable_offset());
+        _partition->probe().update_iceberg_translation_offset_lag(0);
```

Comment:
Is this right? shouldn't this be `read_end_offset`?

Reply:
here we are computing the lag, this branch is taken when there is nothing to translate, so lag = 0?

Reply:
Oh I missed we were directly accessing the probe not calling the helper function. 🙈